tests/test_file.py

```diff
@@ -45,6 +45,26 @@ def test_file_put(self):
         )
         self.assertIn('version', response[2])
 
+    def test_file_put_metadata_from_tags(self):
+        file_uuid = uuid.uuid4()
+        response = self.assertPutResponse(
+            "/v1/files/" + str(file_uuid),
+            requests.codes.created,
+            json_request_body=dict(
+                source_url="s3://hca-dss-test-src/test_bad_source_data/metadata_in_tags",
```

Comment:
I guess it has to be... All right, I need to rework this PR to make it clear that this is the norm.